File: level1/이상한_문자_만들기/Solution.java
class Solution {
    public String solution(String s) {
        char[] cs = s.toCharArray();
        boolean odd = true;
        for (int i = 0; i < s.length(); i++) {
            if (cs[i] == ' ') {
                odd = true;
                continue;
            }
            cs[i] = odd ? Character.toUpperCase(cs[i]) : Character.toLowerCase(cs[i]);
            odd = !odd;
        }
        return String.valueOf(cs);
    }

    public static void main(String[] args) {
        Solution s = new Solution();
        System.out.println(s.solution("try hello world"));
    }
}

File: level1/이상한_문자_만들기/solution.js
function solution(s) {
  s = [...s];
  var odd = true;
  for (var i = 0; i < s.length; i++) {
    if (s[i] === " ") {
      odd = true;
      continue;
    }
    s[i] = odd ? s[i].toUpperCase() : s[i].toLowerCase();
    odd = !odd;
  }
  return s.join("");
}

console.log(solution("try hello world"));
